config/sidekiq.yml

```diff
@@ -17,7 +16,7 @@
 
 :schedule:
   CalculateAllMetrics:
-    cron: '0 9 0 * * *' # Run at 00:09
+    cron: '9 0 * * *' # Run at 00:09
```

Comment:
Sidekiq scheduler uses a 6 slots cron syntax, can you explain what you did there?

Reply:
Found that sidekiq-scheduler uses 5 slots cron syntax [in specs](https://github.com/sidekiq-scheduler/sidekiq-scheduler/blob/7545ded16ec6dbecbbf8c613b49217822a8c39ef/spec/sidekiq-scheduler/scheduler_spec.rb#L476) So I think it is compatible with 6* and 5* format.

I think advantage is that crontab also uses 5* format

Reply:
I didn't know that. I think this is generating a inconstitency with the other scheduling part, I suggest we stay on the 6 stars pattern.